browser/src/GenePage/CopyNumberVariantsInGene.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+
+import { referenceGenome } from '@gnomad/dataset-metadata/metadata'
+import Query from '../Query'
+import { filterCopyNumberVariantsInZoomRegion } from '../RegionViewer/filterVariantsInZoomRegion'
+import CopyNumberVariants from '../CopyNumberVariantList/CopyNumberVariants'
+import { CopyNumberVariant } from '../CopyNumberVariantPage/CopyNumberVariantPage'
+
+type OwnProps = {
+  datasetId: string
+  gene: {
+    chrom: string
+    gene_id: string
+  }
+  zoomRegion?: {
+    start: number
+    stop: number
+  }
+}
+
+// @ts-expect-error TS(2456) FIXME: Type alias 'Props' circularly references itself.
+type Props = OwnProps & typeof CopyNumberVariantsInGene.defaultProps
+
+// @ts-expect-error TS(7022) FIXME: 'CopyNumberVariantsInGene' implicitly has type 'an... Remove this comment to see the full error message
+const CopyNumberVariantsInGene = ({ datasetId, gene, zoomRegion, ...rest }: Props) => {
+  const operationName = 'CopyNumberVariantsInGene'
+  const query = `
+    query ${operationName}($datasetId: CopyNumberVariantDatasetId!, $geneId: String!, $referenceGenome: ReferenceGenomeId!) {
+      gene(gene_id: $geneId, reference_genome: $referenceGenome) {
+        copy_number_variants(dataset: $datasetId) {
+          sc
+          sn
+          sf
+          chrom
+          end
+          filters
+          length
+          pos
+          posmin
+          posmax
+          endmin
+          endmax
+          type
+          variant_id
+        }
+      }
+    }
+  `
+  const url = datasetId === 'gnomad_cnv_r4' ? '/api' : 'https://gnomad.broadinstitute.org/api/'
```

Comment:
Don't forget to remove this bit before merging. When I have to do something like this which is purely to rig something temporarily for a demo, I often do that in a separate commit prefixed with the label `DONTMERGE`, that way I'll see it and remember to drop that commit before I merge to `main`